lib/erlef_web/controllers/blog_controller.ex

```diff
@@ -4,39 +4,37 @@ defmodule ErlefWeb.BlogController do
 
   alias Erlef.Blogs
 
+  # not sure what we want to do with this yet.
   def index(conn, %{"topic" => topic}) do
     with {:ok, posts} <- list(topic) do
-      render(conn, "index.html", topic: topic, posts: posts, about: Blogs.Config.about_for(topic))
+      render(conn, "index.html", topic: topic, posts: posts)
```

Comment:
Correct me if I'm wrong, but this means two working group blogs can not have posts of the same slug. So, if education creates the post with slug "foo", then building can not create a post with slug "foo". Thoughts?

Reply:
that's correct.  my gut says that the likelihood of such conflicts is low enough that we can enforce uniqueness.  i doubt two groups will ever write about exactly the same thing.

this also allows us to use canonical urls like `https://erlef.org/news/some-cool-thing`.  My opinion is that something shorter than `https://erlef.org/news/marketing/some-cool-thing` is preferable, because from a distance, we're all part of the one Foundation.

Reply:
Agreed.